src/app/src/components/EmbeddedMapFieldsConfig.jsx

```diff
@@ -63,6 +66,17 @@ const styles = {
     textInput: {
         width: '250px',
     },
+    searchable: {
+        marginLeft: '15px',
+    },
+    icon: {
+        color: 'rgba(0, 0, 0, 0.38)',
+        verticalAlign: 'super',
+        fontSize: '12px',
+    },
+    tooltip: {
+        fontSize: '16px',
```

Comment:
This will make the tooltip font size match that of the section description (ie, "Choose which fields to display…").
```suggestion
        fontSize: '0.875rem',
```